src/Location/location.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Post, Body, Get, Param, Patch, Delete } from '@nestjs/common';
+import { LocationService } from './location.service';
+import { LocationDto } from './dto/location.dto';
+
+@Controller('location')
+export class LocationController {
+  constructor(private locationService: LocationService) {
+  }
+
+  @Post()
+  addLocation(@Body() locationDto: LocationDto) {
+    this.locationService.insertLocation(locationDto).then();
```

Comment:
what the `then` is used for? if you want to implement an async method, add `async` in the method header and await before the function call.
see what I did in healthEntry.controller.ts

Reply:
thank you for this comment, I didn't know that this is the best practices webstorme offer me to do then this why i did it hahah